### ISSUE TITLE
Add a section about packaging

### ISSUE THREAD

**Issue by Pesc0** [FreeCAD/DevelopersHandbook#50]
As part of the technical documentation it would be great to have a section explaining how the built executable gets packaged for various platforms and OSs.

**Comment by yorikvanhavre** [comment]
Definitely! That's one of the things we want to do with the 0.21 release
